pkgs/tools/misc/xilinx-bootgen/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, openssl }:
+
+stdenv.mkDerivation {
+  pname = "xilinx-bootgen";
+  version = "2019-10-23";
```

Comment:
I couldn't find any official release tag in the repository - 2019.2 seems to be inside an ifdef in `version.h` but that version number hasn't changed since d000746f9621000042d9469686ee867d73321c17.

I'll rename it to `unstable-2019-10-23`, looks safer to me.